client/src/views/workspaces/WorkspacesPage.vue

```diff
@@ -423,6 +430,43 @@ async function openWorkspace(workspace: WorkspaceInfo): Promise<void> {
   }
 }
 
+async function renameWorkspace(workspace: WorkspaceInfo, newName: WorkspaceName): Promise<void> {
+  const result = await parsecRenameWorkspace(newName, workspace.id);
+  if (result.ok) {
+    await refreshWorkspacesList();
+    informationManager.present(
+      new Information({
+        message: translate('WorkspacesPage.RenameWorkspaceModal.success', { newName: newName }),
+        level: InformationLevel.Success,
+      }),
+      PresentationMode.Toast,
+    );
+  } else {
+    informationManager.present(
```

Comment:
`clientRenameWorkspace` can return multiple errors. See
https://github.com/Scille/parsec-cloud/blob/d4fdf118be3c83e19b63750338dd6b79c132aa4c/client/src/plugins/libparsec/definitions.ts#L744

While most of them can be have the same generic message, for some of them it can be helpful to add custom messages.
 - `AuthorNotAllowed` and `Invalid*` can be grouped into a `no permission error`,
 - `Offline` has its own error
 - a generic message should do for the other ones but adding a `console.error` with the tag/error could also help.